FirebaseSessions/Sources/SessionStartEvent.swift

```diff
@@ -256,7 +256,7 @@ class SessionStartEvent: NSObject, GDTCOREventDataObject {
     -> firebase_appquality_sessions_NetworkConnectionInfo_MobileSubtype {
     var subtype: firebase_appquality_sessions_NetworkConnectionInfo_MobileSubtype
 
-    #if os(iOS) && !targetEnvironment(macCatalyst)
+    #if os(iOS) && !targetEnvironment(macCatalyst) && (swift(<5.9) || !os(xrOS))
```

Comment:
Note: This emits the warning `Unknown operating system for build configuration 'os'` in Xcode `<= 14.2`.